external-crates/move/move-execution/v1/crates/move-vm-runtime/src/interpreter.rs

```diff
@@ -242,9 +242,9 @@ impl Interpreter {
                 ExitCode::Call(fh_idx) => {
                     let func = resolver.function_from_handle(fh_idx);
                     // Compiled out in release mode
-                    #[cfg(debug_assertions)]
-                    let func_name = func.pretty_string();
-                    profile_open_frame!(gas_meter, func_name.clone());
+
+                    let _func_name = func.pretty_string();
```

Comment:
nit: you could inline the call into the macro parameter? Then I think we could guarantee that it wouldn't show up in the output if the gas profiler is not enabled.